### ISSUE TITLE
Design Issue : Need to retrieve and show images which are a part of the tweet

### ISSUE THREAD

**Issue by saumyabhadani95** [glciampaglia/infodiversity-mock-social-media#10]
Some tweets have an explicit image. Like the following:

![Screenshot from 2020-12-15 10-38-35](https://user-images.githubusercontent.com/14941766/102236723-d3ebfa80-3ec1-11eb-9882-ca2b11a02473.png)

Right now we are not showing them. So we need to retrieve them, store them and then show them:

![Screenshot from 2020-12-15 10-41-12](https://user-images.githubusercontent.com/14941766/102236930-13b2e200-3ec2-11eb-8d05-e964467caed8.png)

**Comment by RobertAndion** [comment]
It also seems like we are getting the shortened version of the tweet. Are we passing text.full (I forget exact syntax but there is a full version and a shortened one) I have added experimental code to get these images however it is far from perfect in its current state.

**Comment by RobertAndion** [comment]
This is nearly complete. We have it so the first present media is always displayed. We have yet to figure out how to send an array with multiple to be interpreted by java script from python. We also need to decide how to deal with videos and if pug/truman is capable of embedding them (I assume so but it hasnt arrisen). Saumya was going to look into our pipeline issue.

**Comment by glciampaglia** [comment]
We discussed this issue. We decided two things:
1. We will display only the first image. 
2. **If** Pug is able to display the videos natively, then we will have videos, otherwise we won't. This still needs to be tested.

We will revisit the issue after the beta testing phase with users.

**Comment by glciampaglia** [comment]
We implemented 1), but for 2) we discovered that the way videos are rendered in Pug is not very pretty, so we will not have video playback. However, there is not an easy way to filter out tweets with videos, so we will need display them anyway. Twitter will pick a random frame from the video as the image, so that's what they will be displayed with.

We will still keep this issue open until after the beta testing.
